packages/not-next/src/hooks/usePublicClientForRegistry.ts

```diff
@@ -0,0 +1,20 @@
+import { usePublicClient } from "npm:wagmi";
+import * as chains from "npm:wagmi/chains";
+import process from "node:process";
+
+export function usePublicClientForRegistry(id?: number) {
```

Comment:
this function needs cleaned up. The id should be use a default parameter. Something like this
```suggestion
export function usePublicClient(id = chains[process.env.NEXT_PUBLIC_CHAIN_NAME].id) {
```
Also I would rename it to just `usePublicClient` since it is a general hook.

Reply:
I renamed the file to usePublicClient, but since we are importing usePublicClient from wagmi, I changed the function name itself as useClient. But I will still import it as usePublicClient throughout the app since it's not a default export. 
The test for this hook is in my tests branch so i will merge this PR

Reply:
The test for this hook is in my tests branch so i will merge this PR